src/ResourceManagement/Sql/SqlManagement/Microsoft.Azure.Management.Sql.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Management.Sql
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Management.Sql">
-      <PackageVersion>0.54.0-prerelease</PackageVersion>
+      <PackageVersion>0.55.0-prerelease</PackageVersion>

```

Comment:
0.55.0 and 0.55.1 are already being used. Please change to 0.56.0

http://www.nuget.org/packages/Microsoft.Azure.Management.Sql/0.55.1-prerelease